### PR TITLE
add ready buttons to blitz game

### PR DESCRIPTION
**Pull request by PamBWillenz**
- Add columns to game.rb for blitz game ready buttons - white_ready; black_ready
- Add two methods to games#controller - `player_ready`, `player_not_ready`
- Add two match routes to hit with Ajax requests
- Create playerReady.js to store JS for blitz buttons
- Add buttons to games#show - ready, not_ready for black and white players

Very much a work in progress. Need to line up JS with other timer and pusher stuff.

Have not written any tests...but will (at least try :))

### CODE REVIEW

**Comment by PamBWillenz** [comment]
- Added params `white_ready` and `black_ready` to game for tests
- Now getting pusher error that broadcast message is wrong.
- Was able to get timer to start on each player's game but did not broadcast messages

**Comment by PamBWillenz** [comment]
- Changed pusher message to symbol to get messages to broadcast
- refactored the true and save for white and black ready
- changed binds on JS to get them linked to game#controller method
- Matt found/fixed timer_left error while troubleshooting with me
- 
- player_ready timer is working as it should

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -168,9 +212,28 @@ def set_user_color
 
   # Never trust parameters from the scary internet, only allow the white list through.
   def game_params
-    params.require(:game).permit(:current_turn, :white_player_id, :is_blitz, :black_draw, :white_draw, :black_forfeit, :white_forfeit)
-
+    params.require(:game).permit(:current_turn, :white_player_id, :is_blitz, :black_draw, :white_draw, :black_forfeit, :white_forfeit, :has_started, :white_ready, :black_ready)
   end
   
+  def set_game
```
Comment (FsDevNinja) [inline app/controllers/games_controller.rb]: I think these methods are in the application controller.. I think we can get rid of set_game and set_opponent_id

Comment (PamBWillenz) [inline app/controllers/games_controller.rb]: yep - they are in application controller - I pulled from master (twice) and it didn't update my games#controller (those two methods are still there) - so can I just delete them -- without messing anything up in the repo?

Comment (FsDevNinja) [inline app/controllers/games_controller.rb]: yeah.. just delete...

**Pull request merged**